Week-3/Session7-Responsiveness/VacationRespImg/responsiveImg.html

```diff
@@ -29,13 +29,19 @@ <h2>Egypt</h2>
       </p>
 
       <!-- TODO: Add srcset and sizes attributes -->
-      <!-- responsive images is not working  -->
+      <!-- responsive images is not working for default 200px -->
       <img
         src="luxor_600.jpg"
-        alt="Luxor Temple"
-        srcset="luxor_600.jpg 600px, luxor_800.jpg 800px, luxor_1200.jpg 1200px"
-        sizes="(max-width: 500px) 200px, (min-width: 500px) 400px, (min-width: 700px) 600px"
-      />
+       
+        srcset="luxor_600.jpg 600w,
+                 luxor_800.jpg 800w,
+                 luxor_1200.jpg 1200w" 
+         sizes="(min-width: 700px) 600px,
+                (max-width: 500px) 400px,
+                200px" 
+         alt="Luxor Temple">
```

Comment:
is this working now? what was the issue earlier?

Reply:
> is this working now? what was the issue earlier?

yes, earlier I was providing size in px in the srcset, now its adjusting acc. to viewport